Tests/SentryProfilerTests/SentryBacktraceTests.mm

```diff
@@ -199,7 +199,7 @@ - (void)testCollectsMultiThreadBacktrace
     XCTAssertEqual(
         pthread_create(&thread2, nullptr, threadEntry, reinterpret_cast<void *>(bc_d)), 0);
 
-    const auto cache = std::make_shared<ThreadMetadataCache>();
+    ThreadMetadataCache cache;
```

Comment:
Why can't we use `make_unique` here as we do on https://github.com/getsentry/sentry-cocoa/pull/3398/files#diff-0397b590fd7ff98685a61fa9a99cceccf30bdf224754ba5f92448abd0852885bR80

Reply:
Yes, you could do this ^